tests/integration/targets/docker_swarm_service/tasks/tests/mounts.yml

```diff
@@ -418,7 +418,6 @@
         type: "tmpfs"
         tmpfs_size: "50M"
   register: mounts_tmpfs_size_1
-  ignore_errors: yes
```

Comment:
Please keep these, and modify the conditions in the assertions below instead.

Reply:
You're right. Got rid of it.